rpc_jobs/rpc_aio.yml

```diff
@@ -168,6 +168,7 @@
 
     dsl: |
       // CIT Slave node
+      currentBuild.result = 'SUCCESS'
```

Comment:
Jenkins only allows currentBuild.result to change to statuses that are worse than the current status, so we can go from SUCCESS to FAILURE, but not FAILURE to SUCCESS.

https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/model/Run.java#L464